### DISCUSSION TITLE
Nominatim Updater Service: it updates strictly Nominatim DB or also data in PostGIS DB?

### DISCUSSION THREAD

**Discussion by MyckeTap** [osm-search/Nominatim#3348] [Q&A]
Dear Community,
I was wondering whether Nominatim Updater service is doing changes to Nominatim DB only?
Or it also updates record in PostGIS DB?
Cheers,
Ivo

**Comment by mtmail** [comment]
Do you mean https://nominatim.org/release-docs/latest/admin/Update/ ? Nominatim has only one database. It's a Postgresql database with the PostGIS extension enabled.

**Reply by MyckeTap** [reply]
thanks, now I got it.

**Comment by MyckeTap** [comment]
Nominatim updates its own Database only. It's not updating PostGIS standalone DB with data.
